src/views/Books.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-container grid-list-sm fluid>
+  <v-container id="books-container" grid-list-sm fluid>
```

Comment:
i get the usage of id here (for the e2e test), my only problem if we ever reuse it we could potentially have duplicated id's :s, we should create a ticket for studying which solution would fit best atm i'm more inclined for class reference than id's but that's just IMO.

Reply:
Done: https://github.com/orgs/lesfieres/projects/1#card-19548795